exchanges/request/request.go

```diff
@@ -302,14 +302,10 @@ func (r *Requester) DoRequest(req *http.Request, path string, body io.Reader, re
 			}
 		}
 
-		if resp.StatusCode != 200 && resp.StatusCode != 201 && resp.StatusCode != 202 {
-			err = fmt.Errorf("unsuccessful HTTP status code: %d", resp.StatusCode)
-			if verbose {
-				err = fmt.Errorf("%s\n%s", err.Error(),
-					fmt.Sprintf("%s exchange raw response: %s", r.Name, string(contents)))
-			}
-
-			return err
+		if resp.StatusCode < 200 || resp.StatusCode > 299 {
```

Comment:
I think we should stick to 202 being the highest. See here: https://httpstatuses.com/